python/python/lance/arrow.py

```diff
@@ -330,9 +330,9 @@ def pillow_decoder(images):
 
                 from PIL import Image
 
-                return np.stack(
+                return np.stack([
                     Image.open(io.BytesIO(img)) for img in images.to_pylist()
-                )
+                ])
```

Comment:
Is this a lint? I wonder why it doesn't like generators.

Reply:
For numpy + python3.12, this line reports that generator missing a magic method and it can only take list